pkgs/development/libraries/rnnoise/default.nix

```
@@ -1,28 +1,46 @@
-{ stdenv, lib, fetchFromGitHub, autoreconfHook }:
+{ stdenv, lib, fetchurl, fetchzip, autoreconfHook
+, modelUrl ? "" # Allow overriding model URL and hash
+, modelHash ? ""
+}:
 
-stdenv.mkDerivation (rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "rnnoise";
-  version = "2021-01-22";
+  version = "0.2";
 
-  src = fetchFromGitHub {
-    owner = "xiph";
-    repo = "rnnoise";
-    rev = "1cbdbcf1283499bbb2230a6b0f126eb9b236defd";
-    sha256 = "1y0rzgmvy8bf9a431garpm2w177s6ajgf79y5ymw4yb0pik57rwb";
+  src = fetchzip {
+    urls = [
+      "https://gitlab.xiph.org/xiph/rnnoise/-/archive/v${finalAttrs.version}/rnnoise-v${finalAttrs.version}.tar.gz"
+      "https://github.com/xiph/rnnoise/archive/v${finalAttrs.version}.tar.gz"
+    ];
+    hash = "sha256-Qaf+0iOprq7ILRWNRkBjsniByctRa/lFVqiU5ZInF/Q=";
   };
 
+  # Copy from https://gitlab.xiph.org/xiph/rnnoise/-/raw/v${finalAttrs.version}/model_version
+  default_model_version = "0b50c45";
+  default_model_url = "https://media.xiph.org/rnnoise/models/rnnoise_data-${finalAttrs.default_model_version}.tar.gz";
+  default_model_hash = "sha256-SsgcXAiE7EvVkHAmqq4WIJt7ds2df3GvWCCUovmPS0M=";
+
+  model = fetchurl {
+    url = if (modelUrl == "") then finalAttrs.default_model_url else modelUrl;
+    hash = if (modelHash == "") then finalAttrs.default_model_hash else modelHash;
+  };
+
+  patchPhase = ''
```

```suggestion
  postPatch = ''
```

`*Phase` shouldn't be overridden

---

Done. Thanks!